### PR TITLE
Remove echoe and just use bundler

### PR DESCRIPTION
**Pull request by zsyed91**
Since we decided to just use bundler instead of echoe, I went ahead and removed all references to `echoe`. The `Rakefile` no longer builds via echoe.

- Remove the Manifest used by echoe
- Remove echoe dependency and instead use bundler
- Made some very minor cleanups to `gemspec`

### CODE REVIEW

**Comment by zsyed91** [comment]
@elmatou Do these changes look good? I will hold off from merging my PR's for non-trivial stuff so fast :smile:

File: pi_piper.gemspec [hunk 1]
```diff
@@ -24,8 +24,8 @@ Gem::Specification.new do |s|
```
Comment (elmatou) [inline pi_piper.gemspec]: you can also remove or modify :
```ruby
   s.test_files    = s.files.grep(%r{^(test|spec|features)/}) 
   s.rubyforge_project = 'pi_piper'
   s.rubygems_version = '2.2.2'
   s.extra_rdoc_files = # extra mean non rb file only
```

**Comment by elmatou** [comment]
did you check with an interactive session with rubygems ? I would say an update, but... you know...

**Comment by zsyed91** [comment]
what do you mean by interactive session? I did run `gem build pi_piper.gemspec` and it successfully built the gem.

**Comment by elmatou** [comment]
I only meant, did you test it with rubygems server ? like with `rake release`

**Comment by zsyed91** [comment]
Oh I see, would that not actually push it to rubygems? I would assume that if the gem can be built, it should be able to be pushed to rubygems. Unfortunately I did not see a `gem push --dry-run` or `gem push --test` command.

**Comment by zsyed91** [comment]
Having said that, I think we can probably and should do a release as `2.0.0` with your `PWM` changes.

**Comment by elmatou** [comment]
good to me

File: pi_piper.gemspec (hunk 1 above)
Comment (zsyed91) [inline pi_piper.gemspec]: Alright updated `extra_rdoc_files` to only include non-ruby files. And removed they keys you mentions except I kept rubygems_version. I see some other popular gems have it.

**Comment by zsyed91** [comment]
Cool, going to squash the commits and then merge :+1:

**Pull request merged**